test/integration/targets/sentry_project/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+api_token: 7b4aff26d4114b89b322937d5c2ffebaf0180d52b2384a19a072fbefac12f797
```

Comment:
Is it OK for this to be public?

Reply:
This token is hardcoded in the container.
During the tests the container runs in private network.
Isn't it ok?

Reply:
Sounds ok for me.